ArduPlane/tailsitter.cpp

```diff
@@ -250,25 +250,26 @@ bool QuadPlane::tailsitter_transition_fw_complete(void)
     if (roll_cd > 9000) {
         roll_cd = 18000 - roll_cd;
     }
-    if (labs(ahrs_view->pitch_sensor) > tailsitter.transition_angle*100 ||
-        roll_cd > tailsitter.transition_angle*100 ||
-        AP_HAL::millis() - transition_start_ms > uint32_t(transition_time_ms)) {
+    // if roll or pitch exceed transition angle, forwad transition complete
+    if (labs(ahrs_view->pitch_sensor) > tailsitter.transition_angle_fw * 100 || roll_cd > tailsitter.transition_angle_fw * 100) {
```

Comment:
```suggestion
    if (labs(ahrs_view->pitch_sensor) > tailsitter.transition_angle_fw * 100 || labs(roll_cd) > tailsitter.transition_angle_fw * 100) {
```

Not a issue from this PR but we should fix